packages/app/navigation/index.tsx

```diff
@@ -62,6 +64,13 @@ export function NavigationProvider({
         formatter: (options) =>
           options?.title ? `${options.title} | Showtime` : "Showtime",
       }}
+      onStateChange={() => {
+        // this is a temporary workaround to reduce crashes.
+        // Once we have optimized images and pull from disk, we can remove this
+        // Currently we're collecting images into memory until the app crashes,
+        // therefore we want to clear it on every screen change
+        Image.clearMemoryCache();
+      }}
```

Comment:
First, I've registered a `setInterval` but I think clearing the memory on navigation state chage is the better solution. This will clear the memory when navigating away from the screen and reduces crashes.